deps/jerry/jerry-core/lit/lit-magic-strings.inc.h

```diff
@@ -528,8 +533,14 @@ LIT_MAGIC_STRING_DEF (LIT_MAGIC_STRING_SET_UTC_MILLISECONDS_UL, "setUTCMilliseco
 LIT_MAGIC_STRING_DEF (LIT_MAGIC_STRING_TO_LOCALE_DATE_STRING_UL, "toLocaleDateString")
 LIT_MAGIC_STRING_DEF (LIT_MAGIC_STRING_TO_LOCALE_TIME_STRING_UL, "toLocaleTimeString")
 #endif
+#if  defined (JERRY_HEAP_PROFILER)
+LIT_MAGIC_STRING_DEF (LIT_MAGIC_STRING_EXTERNAL_FUNCTION, "<external function>")
+#endif
 LIT_MAGIC_STRING_DEF (LIT_MAGIC_STRING_GET_OWN_PROPERTY_NAMES_UL, "getOwnPropertyNames")
 LIT_MAGIC_STRING_DEF (LIT_MAGIC_STRING_PROPERTY_IS_ENUMERABLE_UL, "propertyIsEnumerable")
+#if defined (JERRY_HEAP_PROFILER)
+LIT_MAGIC_STRING_DEF (LIT_MAGIC_STRING_LEX_ENV, "<lexical environment>")
+#endif
```

Comment:
Why not grouping these `JERRY_HEAP_PROFILER` magic string together?

Reply:
All code style problem in this file should be ignored.